tests/bson/bson-int64-001.phpt

```diff
@@ -0,0 +1,46 @@
+--TEST--
+MongoDB\BSON\Int64 roundtripped through BSON on 32-bit platforms
+--SKIPIF--
+<?php if (4 !== PHP_INT_SIZE) { die('skip Only for 32-bit platform'); } ?>
+--FILE--
+<?php
+
+require_once __DIR__ . '/../utils/tools.php';
+
+/* Omit testing values in the range of 32-bit integers. They will come back from
+ * BSON as a PHP integer type, as Int64 is only used when necessary. */
+$tests = [
+    (object) ['int64' => unserialize('C:18:"MongoDB\BSON\Int64":43:{a:1:{s:3:"int";s:19:"9223372036854775807";}}')],
+    (object) ['int64' => unserialize('C:18:"MongoDB\BSON\Int64":44:{a:1:{s:3:"int";s:20:"-9223372036854775808";}}')],
```

Comment:
Please add test cases that covers the two "just out of 32-bit range" cases as well.